pydrad/configure/configure.py

```diff
@@ -200,14 +203,16 @@ def setup_hydrad(self, root_dir):
         else:
             build_script = 'build_HYDRAD.bat'
         # NOTE: make sure we have needed permissions to run compile script
+		# Only do this on Unix-based systems (chmod not valid on Windows)
+        if not self.on_windows:
+            run_shell_command(
+                ['chmod', 'u+x', build_script],
+                os.path.join(root_dir, 'HYDRAD/build_scripts'),
+                shell=False,
+            )
         run_shell_command(
-            ['chmod', 'u+x', build_script],
-            os.path.join(root_dir, 'HYDRAD/build_scripts'),
-            shell=False,
-        )
-        run_shell_command(
-            [f'./{build_script}'],
-            os.path.join(root_dir, 'HYDRAD/build_scripts'),
+			[f'./{build_script}'],
+			os.path.join(root_dir, 'HYDRAD/build_scripts'),
```

Comment:
And here too
```suggestion
		[f'./{build_script}'],
		os.path.join(root_dir, 'HYDRAD/build_scripts'),
```

Reply:
Probably because of the editor I was using, which prefers tabs to four spaces.